### ISSUE TITLE
Aggregate uuid as integer

### ISSUE THREAD

**Issue by mpran** [commanded/commanded#187]
Hi. Awesome library, but getting an exception if I try to use an integer for aggregate id.
Is there a reason why I shouldn't use an integer?

sample code in router:
`some_id: 1`
```
identify SomeAggregate, by: :some_id, prefix: ""
```

```
** (ArgumentError) argument error
    :erlang.bit_size(1)
    (commanded) lib/commanded/middleware/extract_aggregate_identity.ex:49: Commanded.Middleware.ExtractAggregateIdentity.prefix/2
    (commanded) lib/commanded/middleware/extract_aggregate_identity.ex:20: Commanded.Middleware.ExtractAggregateIdentity.before_dispatch/1
    (commanded) lib/commanded/middleware/pipeline.ex:109: Commanded.Middleware.Pipeline.chain/3
    (commanded) lib/commanded/commands/dispatcher.ex:45: Commanded.Commands.Dispatcher.dispatch/1
```

Tracked it down to this line https://github.com/commanded/commanded/blob/266587f7786d27b390a355991cbb792944ac0885/lib/commanded/middleware/extract_aggregate_identity.ex#L49

Changing it to `do: "#{identity_prefix}#{aggregate_uuid}"` works fine

Is there something I'm missing or setting up my router incorrectly?

**Comment by slashdotdash** [comment]
The aggregate identity is assumed to be a string, so it won't work if you provide an integer as the identity. This needs to be better handled and documented by Commanded. There's a relevent a open pull request to allow the aggregate uuid to be of any type that implements the `String.Chars` protocol (#166).

**Comment by Freyskeyd** [comment]
I think this issue is resolved now.

**Comment by slashdotdash** [comment]
Fixed by #166.
